docs/language/builtins.rst

```diff
@@ -49,8 +49,8 @@ uint128 ``msg.value``
 bytes ``msg.data``
     The raw ABI encoded arguments passed to the current call.
 
-bytes4 ``msg.sig``
-    Function selector from the ABI encoded calldata, e.g. the first four bytes. This
+bytes4 (Substrate) or bytes8 (Solana) ``msg.sig``
+    Function selector from the encoded calldata, e.g. the first four or eight bytes. This
```

Comment:
This should have a note saying that the selector is know as a discriminator on Solana.